ts/features/fastLogin/store/reducers/securityAdviceReducer.ts

```diff
@@ -0,0 +1,43 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { PersistConfig, persistReducer } from "redux-persist";
+import { getType } from "typesafe-actions";
+import { Action } from "../../../../store/actions/types";
+import { setSecurityAdviceAcknowledged } from "../actions/securityAdviceActions";
+
+export type SecurityAdviceAcknowledgedState = {
+  acknowledged: boolean | undefined;
+};
+
+export const securityAdviceAcknowledgedInitialState: SecurityAdviceAcknowledgedState =
+  {
+    acknowledged: false
+  };
+
+const securityAdviceAcknowledgedReducer = (
+  state: SecurityAdviceAcknowledgedState = securityAdviceAcknowledgedInitialState,
+  action: Action
+): SecurityAdviceAcknowledgedState => {
+  switch (action.type) {
+    case getType(setSecurityAdviceAcknowledged):
+      return {
+        ...state,
+        acknowledged: true
+      };
+    default:
+      return state;
+  }
+};
+
+const CURRENT_REDUX_FEATURES_STORE_VERSION = -1;
```

Comment:
I had noticed this too, but I don't know what the advantage might be in changing the name but letting the variable do the same thing and i think it would be better to keep it with the same name to maintain the naming convention. Anyway, there is no problem to changing the name!